### ISSUE TITLE
Input field billing_address_2 is missing a label

### ISSUE THREAD

**Issue by rianrietveld** [woocommerce/woocommerce#21182]
**Describe the bug**
For a form to work properly on every device, there should be a label attached to every input field. This way the input field can be properly targeted and announced by assistive technology like a screen reader.
In the Check out form, with the street address, there are two input fields with one label.
In the generated HTML:

```
<p class="form-row form-row-wide address-field validate-required" id="billing_address_1_field" data-priority="50">
  <label for="billing_address_1" class="">
    Street address&nbsp;<abbr class="required" title="required">*</abbr>
  </label>
  <span class="woocommerce-input-wrapper">
    <input class="input-text " name="billing_address_1" id="billing_address_1" placeholder="House number and street name" value="my street" autocomplete="address-line1" type="text">
  </span>
</p>
<p class="form-row form-row-wide address-field" id="billing_address_2_field" data-priority="60">
  <span class="woocommerce-input-wrapper">
    <input class="input-text " name="billing_address_2" id="billing_address_2" placeholder="Apartment, suite, unit etc. (optional)" value="my unit number" autocomplete="address-line2" type="text">
  </span>
</p>
```
There is a label for `billing_address_1` but not for `billing_address_2`. 
This makes the form not pass WCAG 2 AA accessibility standards.

See also: class WC_Countries get_default_address_fields()
There is no label defined for `address_2`

**To Reproduce**
1. Go to the checkout view
2. Click on inspect code
3. Run [aXe](https://www.deque.com/axe/)
4. See error ["Ensures every form element has a label", critical](https://dequeuniversity.com/rules/axe/3.0/label)

**Expected behavior**
One solution can be:
Because the Woo CSS already has the .screen-reader-text class, an extra label can be added, invisible on screen, but announced for a screen reader.
Like for example:
```
<p class="form-row form-row-wide address-field validate-required" id="billing_address_1_field" data-priority="50">
  <label for="billing_address_1" class="">
    Street address&nbsp;<abbr class="required" title="required">*</abbr>
  </label>
  <span class="woocommerce-input-wrapper">
    <input class="input-text " name="billing_address_1" id="billing_address_1" placeholder="House number and street name" value="my street" autocomplete="address-line1" type="text">
  </span>
</p>
<p class="form-row form-row-wide address-field" id="billing_address_2_field" data-priority="60">
  <label for="billing_address_2" class="screen-reader-text">
    Apartment, suite, unit etc. (optional)
  </label>
  <span class="woocommerce-input-wrapper">
    <input class="input-text " name="billing_address_2" id="billing_address_2" placeholder="Apartment, suite, unit etc. (optional)" value="my unit number" autocomplete="address-line2" type="text">
  </span>
</p>
```
**Isolating the problem (mark completed items with an [x]):**
- [x] I have deactivated other plugins and confirmed this bug occurs when only WooCommerce plugin is active.
- [x] This bug happens with a default WordPress theme active, or [Storefront](https://woocommerce.com/storefront/).
- [x] I can reproduce this bug consistently using the steps above.

**Comment by claudiulodro** [comment]
See #21193. Thanks for the report! Always good to have an expert analyze our accessibility.

**Comment by rianrietveld** [comment]
@claudiosan @rodrigoprimo Thanks for the fast respons and fix <3

**Comment by ChristophAnastasiades** [comment]
What is the reason for the "billing_address_2_field" to be visible only to screen readers? Why not make it visible to visitors just like the "billing_address_1_field" label?
